webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
- Then both fingers release from the touchscreen.
-
-<p>To execute these actions,
- we must send the "actions" endpoint two dictionary objects
- in the JSON list of dictionaries, one for each finger.
- We must use the "id" key of each object to uniquely identify each finger.
- The "actions" key will hold all the actions the input source will take.
-
-<p>The JSON for this set of actions is as follows:
-
-<p><pre>[
-  {
-    "source": "touch",
-    "id": "1",
-    "actions": [
-              { "name": "pointerDown",
-                "ELEMENT": "element1"
-              },
-              { "name": "pause",
-                "duration": 0
-              },
-              { "name": "pointerUp"
-              }
-    ]
-  },
-  {
-    "source": "touch",
-    "id": "2",
-    "actions": [
-              { "name": "pointerDown",
-                "ELEMENT": "element2"
-              },
-              { "name": "pointerMove",
-                "ELEMENT": "element3"
-              },
-              { "name": "pointerUp"
-              }
-    ]
-  }
-]</pre>
-
-<p>When the remote end receives this,
- it will look at each input source's action lists.
- It will dispatch the first action of each source together,
- then the second actions together, and lastly, the final actions together.
-
-<p>The diagram below displays when each action gets executed.
- "Source 1" is the first finger, and "source 2" is the second.
-
-<p><img src=graphics/note1actions.svg alt="">
-
-<p>There is no limit to the number of input sources,
- and there is no restriction regarding the length of each input's action list.
- Meaning, there is no requirement that all action lists have to be the same length.
- It is possible for one input source's action list
- may have more actions than another.
- As an example, imagine having two fingers on a touchscreen.
- The first finger will press on element1 while the second presses on element2,
- then the first will release the touchscreen while the second finger moves to element3,
- and finally the second finger releases from the touchscreen.
- In this case, the action list for the first finger contains 2 actions (pointerDown, pointerUp),
- and the action list for the second finger contains 3 (pointerDown, pointerMove, pointerUp).
- In this case, the JSON will look like this:
-
-<p><pre>[
-  {
-    "source": "touch",
-    "id": "1",
-    "actions": [
-              { "name": "pointerDown",
-                "ELEMENT": "element1"
-              },
-              { "name": "pointerUp"
-              }
-    ]
-  },
-  {
-    "source": "touch",
-    "id": "2",
-    "actions": [
-              { "name": "pointerDown",
-                "ELEMENT": "element2"
-              },
-              { "name": "pointerMove",
-                "ELEMENT": "element3"
-              },
-              { "name": "pointerUp"
-              }
-    ]
-  }
-]</pre>
+    <p>The <dfn>Add Actions</dfn> <a>command</a> allows you to create <dfn>sequential
```

Comment:
I think we should find a better term than _Add Actions_.  Maybe _Perform Actions_ or just _Actions_?